source/tasks/DeployTenant/TenantedDeployV6/task.json

```diff
@@ -0,0 +1,126 @@
+﻿{
+    "id": "a847e2d1-5435-4d52-a774-6d300953e85f",
+    "name": "OctopusDeployReleaseTenanted",
+    "friendlyName": "Deploy Octopus Release to Tenants",
+    "description": "Deploy an Octopus Deploy Release to Tenants",
+    "helpMarkDown": "set-by-pack.ps1",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "author": "Octopus Deploy",
+    "version": {
+        "Major": 6,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "demands": [],
+    "minimumAgentVersion": "2.144.0",
+    "groups": [
+        {
+            "name": "tenant",
```

Comment:
feels like this probably isn't needed for this step. The groups are good for collapsing optional/advanced things out of the way, but Tenants are required for this step so should always be visible.

Reply:
group removed, ui screenshot updated